tests/test_table.py

```diff
@@ -148,12 +148,12 @@ def test_table_display_result_null(db: gp.Database):
     # fmt: on
     t = gp.to_table(rows, db=db, column_names=["id", "animal"])
     expected = (
-        "| id     || animal |\n"
-        "====================\n"
-        "| [1]    ||        |\n"
-        "| [2]    || Tiger  |\n"
-        "| [3]    ||        |\n"
-        "| [None] || Fox    |\n"
+        " id     | animal \n"
```

Comment:
Can we have a list which has multiple elements in this test?

Reply:
Sure